task-2/src/components/AddMovieForm/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { Dropdown } from '../Dropdown';
+import PropTypes from 'prop-types';
+
+import './index.css';
+
+export const AddMovieForm = ({
+	dropdownOptions,
+	dropdownValue,
+	dropdownHandler,
+}) => {
+	const [values, setValues] = useState({
+		title: '',
+		year: '',
+		rating: '',
+		runtime: '',
+		genre: '',
+		description: '',
+	});
+
+	const [selectedDate, setSelectedDate] = useState(null);
+	const handleDateChange = (date) => setSelectedDate(date);
+
+	const handleInputChanges = (e) => {
+		const name = e.target.name;
+		const value = e.target.value;
+		setValues((values) => ({ ...values, [name]: value }));
+	};
+
+	// Still need to finish submit function
+	const handleSubmit = (e) => {
+		e.preventDefault();
+	};
+
+	const resetInputValues = () => {
+		setValues('');
+	};
+
+	return (
+		<>
+			<form onSubmit={handleSubmit} className='formContainer'>
+				<input
+					type='text'
+					className='inputField'
+					placeholder='title'
+					name='titleName'
+					values={values.title}
+					onChange={handleInputChanges}
+				/>
+				<DatePicker
+					selected={selectedDate}
+					onChange={handleDateChange}
+					dateFormat='yyyy'
+					name='selectDate'
+					showYearDropdown
+					scrollableYearDropdown
+					isClearable
+				/>
+				<input
+					type='number'
+					className='inputField'
+					placeholder='rating'
+					name='ratingName'
+					values={values.rating}
+					onChange={handleInputChanges}
+				/>
+				<Dropdown
+					dropdownOptions={dropdownOptions}
+					dropdownValue={dropdownValue}
+					onChange={dropdownHandler}
+				/>
+				<input
+					type='number'
+					className='inputField'
+					placeholder='runtime'
+					name='runtimeName'
+					values={values.runtime}
+					onChange={handleInputChanges}
+				/>
+				<input
+					type='text'
+					className='inputField'
+					placeholder='description'
+					name='descriptionName'
```

Comment:
Consider putting such change-sensitive and potentially repeating values into separate constants.
You can aggregate such values in an object with appropriate name outside of the component body.
E.g. `const ADD_FIELDS = { ..., DESCRIPTION_NAME = 'descriptionName', ...};`
And then here in `AddMovieForm` you'll have the following:
`<input ... name={ADD_FIELDS.DESCRIPTION_NAME} ... />`

Reply:
Consider creating a "constants" folder under "src/core" where all the main reusable non-react things will be stored.

Reply:
"constants" will hold any number of file to your liking - as long as they contain all the hard-coded numbers/strings that you use throughout the app. The idea is to have a central place for all the constants to be able to easily reuse/change them.

Reply:
Once all the constants are in "src/core/constants" don't forget to go through the app and use them where applicable.

Reply:
- Constants folder has been created. 
- All reusable values have been moved into this folder.
- Form input "name" has been changed to match the dynamic nature of using constants.